_component/tabs/index.md

```diff
@@ -18,14 +18,22 @@ iframe_height: medium
 
 {% include partials/html-component.md %}
 
-<h3>SCSS <span class="link"><a href="scss/component.scss" target="_blank">Download SCSS</a></span></h3>
+<h3>PostCSS <span class="link"><a href="scss/component.scss" target="_blank">Download PostCSS</a></span></h3>
```

Comment:
@colorful-tones @garand linking to an scss directory and scss file

Reply:
I added a new `_includes/partials/css-component.md` template and updated all links to from SCSS to CSS